### ISSUE TITLE
v0.12.0  Library isn't fully showing up

### ISSUE THREAD

**Issue by JoshuaPettus** [owncloud/music#733]
Hi,

Since I updated to version 0.12.0 my library isn't displaying fully anymore.  I have about 2000 songs in my library.  I see all the folders, for instance but none of the tracks past the first 5 folders are shown.  Something similar is happening in Album view as well, and I only see the album cover and tracks of the first 5 albums.  After that, only the album titles.

I've tried deleting and re-scanning the library to no luck.

I'm using Firefox 71.0 and Nextcloud 17.0.2

Thank-you!

**Comment by paulijar** [comment]
Arrrgh, I did extensive testing of the scrolling on Nextcloud 13 and ownCloud 10, and testing of other things on Nextcloud 17. But my test collection on NC17 was so small that no much scrolling happened. I should have known, that there is a significant difference on the scrolling containers between *< NC14* and *>= NC14* but somehow this slipped my mind during implementation and testing.

Anyway, I now released a quick fix v0.12.1. It should solve the issue. Thanks for the report!
